ruby/lib/ci/queue/redis/base.rb

```diff
@@ -5,6 +5,7 @@ module Redis
       class Base
         include Common
 
+        ONE_HOUR = 60 * 60
```

Comment:
One hour seems like a lot given that the default expiry is something like 8 hours. I think 10 minutes would be plenty.